src/main/java/org/traccar/web/client/model/api/IDevicesService.java

```diff
@@ -107,6 +136,13 @@ public EditDeviceDto(Device device) {
             this.arrowStoppedColor = device.getIconArrowStoppedColor();
             this.arrowPausedColor = device.getIconArrowPausedColor();
             this.arrowOfflineColor = device.getIconArrowOfflineColor();
+            
+            this.maintenances = new ArrayList<>();
+            for(Maintenance m : device.getMaintenances())
```

Comment:
Czy ```device.getMaintenances()``` nie może być nullem? W kmonstruktorze Device może to pole nie zostać ustawione.